src/main/java/com/yahoo/sketches/theta/UpdateSketchComposition.java

```diff
@@ -0,0 +1,105 @@
+package com.yahoo.sketches.theta;
+
+import com.yahoo.memory.WritableMemory;
+import com.yahoo.sketches.Family;
+import com.yahoo.sketches.ResizeFactor;
+
+/**
+ * @author eshcar
+ */
+public class UpdateSketchComposition extends UpdateSketch {
```

Comment:
This indirection is unfortunate as this will likely be a performance hit on a single thread 1:1 comparison with the non-concurrent sketch. You make it up with parallel threads of course.

Nonetheless, we need some documentation to give guidance what the trade-offs are so users don't unwittingly always throw in the concurrent version when it is not needed.

Reply:
I agree

Reply:
If we restrict the concurrent sketch to only QuickSelect theta sketches can we eliminate this?